src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+
+    <p className="Person__partner">
+      {person.sex === 'f' &&
+        person.isMarried === true &&
+        `${person.partnerName} is my husband`}
+      {person.sex === 'm' &&
+        person.isMarried === true &&
+        `${person.partnerName} is my wife`}
+      {person.isMarried === false && `I am not married`}
```

Comment:
try to simplify this solution.
- do not compare with boolean
- make two ternary expressions in component: first will be deside wife or husband should be rendered (let's name it partnerType) and second will be deside about final text: it will be `${person.partnerName} is my ${parnerType}` or `I am not married`